source/_components/weather.buienradar.markdown

```diff
@@ -22,10 +22,13 @@ To add the buienradar weather to your installation, add the following to your `c
 # Example configuration.yaml entry
 weather:
   - platform: buienradar
+  - name: buienradar
```

Comment:
Since the name is an optional parameter is shouldn't be included in the most basic configuration example.